synapse/storage/databases/main/room.py

```diff
@@ -713,6 +715,11 @@ async def get_retention_policy_for_room(self, room_id: str) -> Dict[str, int]:
         Returns:
             A dict containing "min_lifetime" and "max_lifetime" for this room.
         """
+        # If the room retention feature is disabled, return a policy with no minimum nor
+        # maximum, in order not to filter out events we should filter out when sending to
+        # the client.
```

Comment:
I'm finding the second half of this comment difficult to comprehend. Maybe the following makes sense? I'm not 100% sure it is accurate.

```suggestion
        # If the room retention feature is disabled, return a policy with no minimum nor
        # maximum. This prevents incorrectly filtering out events when sending to
        # the client.
```